app/pages/sections/additional-services/select/recipient/controller.test.js

```diff
@@ -0,0 +1,24 @@
+import {
+  getAdditionalServiceRecipientPageContext,
+} from './controller';
+import * as contextCreator from './contextCreator';
+
+jest.mock('buying-catalogue-library');
+
+jest.mock('./contextCreator', () => ({
+  getContext: jest.fn(),
+}));
+
+describe('Additional-services select-recipient controller', () => {
+  describe('getRecipientPageContext', () => {
+    it('should call getContext with the correct params', async () => {
+      contextCreator.getContext
+        .mockResolvedValueOnce();
+
+      await getAdditionalServiceRecipientPageContext({ orderId: 'order-1', solutionName: 'Solution One' });
```

Comment:
can we change solutionName to itemName please. Also the test data `Solution One` to `Additional Service One`